shared_bootstrap.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -e
+set -eo pipefail
```

Comment:
We might want to _open_ with `set e`, and then add a `set -o pipefail` *after* [line 5](https://github.com/unitasglobal/nio-bootstrap/pull/8/files#diff-708013fcbf80c293a6e806746dde679a0de1d3d5bd42cf932fa3504c01c3dad3R5), so that when we run the `awscli` check in a case where it isn't yet installed, the script continues on to install it rather than failing early.